docs/mnist_example/README.md

```diff
@@ -198,13 +198,13 @@ int main() {
 
   // Create a tensor using omTensorCreateWithOwnership (returns a pointer to OMTensor).
   // When the parameter, owning is set to "true", the OMTensor will release the data
-  // pointer upon destruction. If owning is set to false, the data pointer will not be
+  // pointer upon destruction. If owning is set to false, the tensor array will not be
```

Comment:
A couple of notes:
1. in the same sentence, one part has "data pointer" and another has "tensor array". These should be the same
1. OMTensor itself has a paramater called "data_ptr" so I think either "data_ptr" or "data pointer" is fine here.

Reply:
Thanks!

Reply:
Okay this is fixed